js/market.js

```diff
@@ -1,53 +1,63 @@
 import {url} from './urlService.js';
-const urlEndPoint= url();
 
-$.get(`${urlEndPoint}/plans`)
+const urlEndPoint = url();
+
+$.get(`${urlEndPoint}/plan-management/users/galorstudent@gmail.com/plans`)
     .done((result) => {
 
         // Create container
         let container = document.getElementsByClassName('container')[0];
 
         // Create card for each plan
-        for (let i in result) {
+        for (let i in result.plans) {
+
+            // shortcut for exact plan in the loop
+            const currentPlan = result.plans[i]

```

Comment:
It doesn't make the code cleaner or more readable in my opinion, same same, isn't it ?